wizards/openacademy_wizard.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields
+
+class Wizard(models.TransientModel):
+    _name = "openacademy.wizard"
+    _description = "Openacademy Wizard"
+
+    session_id = fields.Many2one("openacademy.session", required=True)
+
+    partner_ids = fields.Many2many(
+        comodel_name="res.partner",
+        relation="openacademy_wizard_partner_rel",
+        column1="wizard_id",
+        column2="partner_id",
+        string="Partners",
+    )
+
```

Comment:
Remove extra blank line at end of file.

I'd recommend you run `flake8` before pushing to avoid this. For instance, I just ran it over this branch and it output the following:

```
./models/openacademy_session.py:4:1: E302 expected 2 blank lines, found 1
./models/openacademy_session.py:38:47: E226 missing whitespace around arithmetic operator
./models/openacademy_session.py:38:68: E226 missing whitespace around arithmetic operator
./models/res_partner.py:3:1: E302 expected 2 blank lines, found 1
./models/openacademy_course.py:3:1: E302 expected 2 blank lines, found 1
./wizards/openacademy_wizard.py:3:1: E302 expected 2 blank lines, found 1
./wizards/openacademy_wizard.py:16:1: W391 blank line at end of file
```

Could you fix the wizard-related ones here and create a new PR to fix lints from other code, please?